src/components/ApplicationGroup/Details/EnvironmentOverview/HibernateModal.tsx

```diff
@@ -0,0 +1,92 @@
+import { VisibleModal, showError } from '@devtron-labs/devtron-fe-common-lib'
+import React, { useState } from 'react'
+import { ReactComponent as HibernateModalIcon } from '../../../../assets/icons/ic-medium-hibernate.svg'
+import { hibernate } from './service'
+import { ButtonWithLoader } from '../../../common'
+
+interface HibernateModalProps {
+    selectedAppIds: number[]
+    envName: string
+    envId: string
+    setOpenHiberateModal: (value: boolean) => void
+    setAppStatusResponseList: React.Dispatch<React.SetStateAction<any[]>>
+    setShowHibernateStatusDrawer: React.Dispatch<
+        React.SetStateAction<{
+            hibernationOperation: boolean
+            showStatus: boolean
+        }>
+    >
+}
+
+export const HibernateModal = ({
```

Comment:
Same comments as Unhibernate modal.

Suggestion (skip implementing in this PR): Both of the modals can be abstracted to a single component